clearpath_computer_installer.sh

```diff
@@ -234,7 +234,23 @@ step_setup_osrf_packge_server() {
 
     # See https://docs.ros.org/en/jazzy/Installation/Ubuntu-Install-Debs.html
     export ROS_APT_SOURCE_VERSION=$(curl -s https://api.github.com/repos/ros-infrastructure/ros-apt-source/releases/latest | grep -F "tag_name" | awk -F\" '{print $4}')
-    curl -L -o /tmp/ros2-apt-source.deb "https://github.com/ros-infrastructure/ros-apt-source/releases/download/${ROS_APT_SOURCE_VERSION}/ros2-apt-source_${ROS_APT_SOURCE_VERSION}.$(. /etc/os-release && echo $VERSION_CODENAME)_all.deb"
+
+    if [ -z "${ROS_APT_SOURCE_VERSION}" ]; then
+      # some users may encounter this error with the above:
+      #    "API rate limit exceeded for <IP>. [...]"
+      # If this happens, just use a known version.
+      log_warn "Failed to fetch latest apt-source version from GitHub. Falling back to 1.1.0"
+      export ROS_APT_SOURCE_VERSION="1.1.0"
```

Comment:
Can't the current tag be queried and then used?

Reply:
Not easily; to query the current tag we'd need to ask GitHub, but if we're being rate-limited I'm not sure that would actually be any more reliable; the request to query the current tag would probably be limited the same way as the initial request was.

Reply:
Good call. I've modified the code to use this command instead. I think it's worth keeping the error-handling/fallback option for now, just in case there are issues. But this should solve the immediate issue.